core/test/unit/middleware/spam-prevention_spec.js

```diff
@@ -19,6 +20,7 @@ describe('Middleware: spamPrevention', function () {
         spyNext = sinon.spy(function (param) {
             error = param;
         });
+        middleware.spamPrevention = rewire('../../../server/middleware/spam-prevention').__get__('spamPrevention');
```

Comment:
you're right.  `__get__('spamPrevention')` is unnecessary, rewire the whole file just work. I have deleted `__get__('spamPrevention')` and rebased my branch to clean-up history. Thank you for pointing out.